app/models/project.rb

```diff
@@ -17,7 +17,7 @@ class Project < ActiveRecord::Base
 
   accepts_nested_attributes_for :project_contents
 
-  validates_inclusion_of :private, in: [true, false]
+  validates_inclusion_of :private, :live, in: [true, false]
```

Comment:
I've added null: false to the column definition however we still need the validation for useful error messages. If live gets set via `project.update_attributes(live: nil)` you will get a `ActiveRecord::StatementInvalid:PG::NotNullViolation: ERROR`